docs/restore-from-dump.md

```diff
@@ -0,0 +1,141 @@
+# Restore Stelace API data into own infrastructure
+
+Prerequisite:
+- Fulfill the .env file as indicating in README.md
+- Start your databases and Stelace API server
+- Have the Stelace dump
```

Comment:
`Get your PostgreSQL database dump. If hosted in SaaS, get in touch with support to have your database exported.`

I would move this one to top position in the list.